Cannot find module when run nodebb.bat

- **NodeBB version:** master
- **NodeBB git hash:** c831ff0de39a0cd731e29abf633999e4ed990609
- **Database type:** mongo, redis, or postgres
- **Database version:** 
- **Exact steps to cause this issue:**
on windows.
```
git clone https://github.com/NodeBB/NodeBB.git
./nodebb.bat setup
```
- **What happened instead:**
```
PS D:\workspace\NodeBB> ./nodebb.bat
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module './package-install'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Module.require.main.require (D:\workspace\NodeBB\require-main.js:8:10)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (D:\workspace\NodeBB\src\cli\index.js:7:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
```

--- a/nodebb.bat
+++ b/nodebb.bat
@@ -1,1 +1,1 @@
-@echo off && cd %~dp0 && node ./src/cli %*
+@echo off && cd %~dp0 && node ./nodebb %*

